src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -2266,4 +2247,47 @@ public function should_queue_import( $should_queue_import = null ) {
 
 		$this->update_meta( $key, (bool) $should_queue_import );
 	}
+
+	/**
+	 * Attaches a service-provided image to an organizer.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $organizer_id The organizer post ID.
+	 * @param string $service_image
+	 * @param \Tribe__Events__Aggregator__Record__Activity $activity
```

Comment:
No need to change it, but we don't really need a fully qualified class name here and could drop the leading `\`